modules/n1ql/pages/n1ql-language-reference/insert.adoc

```diff
@@ -1280,13 +1289,12 @@ EXPLAIN INSERT INTO `travel-sample` (key UUID(), value _airport)
           "#operator": "Sequence",
           "~children": [
             {
-              "#operator": "IntersectScan", // <3>
+              "#operator": "IntersectScan",
```

Comment:
Make sure to add the callout numbers back into the examples when you update them
```suggestion
              "#operator": "IntersectScan", // <3>
```